packages/components/src/components/entities/actions.ts

```diff
@@ -152,7 +152,7 @@ function resolveEntityParentTypeFromIds(schemaQuery: SchemaQuery, response: any)
     ]);
 }
 
-function extractEntityTypeOptionFromRow(row: Map<string, any>): IEntityTypeOption {
+export function extractEntityTypeOptionFromRow(row: Map<string, any>): IEntityTypeOption {
```

Comment:
I hope that we might not need to export this method in the end, but I see it makes sense while the sample creation stuff is still in transition.

Reply:
There are a few like this that will hopefully be factored into shared components only